integration/docker/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 1 ]]; then
+  echo 'expected one argument: "master", "worker", or "proxy"'
+  exit 1
+fi
+
+service=$1
+
+home=/opt/$(ls /opt | grep alluxio)
+cd ${home}
+
+for keyvaluepair in $(env | grep "ALLUXIO_"); do
```

Comment:
The environment variables are still picked up, just not copied to `alluxio-site.properties`. I've tested that I can use `ALLUXIO_RAM_FOLDER` to configure the ramdisk path, and with this blacklist it doesn't end up in `alluxio-site.properties`. Currently we ignore any unrecognized properties in `alluxio-site.properties`, so it wouldn't be a disaster to copy it, but it could be confusing to the user and we may some day print warnings for unrecognized properties.